.github/workflows/test.yml

```diff
@@ -0,0 +1,26 @@
+name: Test
+on: [push, pull_request]
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: [12, 14]
+    name: Node ${{ matrix.node }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Use node ${{ matrix.node }}
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node }}
+      - name: Install
+        run: npm install
+      - name: Test
+        run: npm run test-browsers
```

Comment:
there is a single `npm test` at the repo, not sure which one is wished here.

Reply:
You picked the right one! The `test-browsers` script is for CI while `test` is for local use.